kapitan/cli.py

```diff
@@ -196,7 +197,17 @@ def main():
         parser.print_help()
         sys.exit(1)
 
+    if hasattr(args, 'verbose') and args.verbose:
```

Comment:
It's a bit more pythonic to do `if 'verbose' in args`

Reply:
That would check if 'verbose' is in the args dictionary no? 
I want to check if the object args, has the attribute .verbose first because it might not be defined in the args parsed.
```
python -i
class Object(object):
    pass
a = Object()
a.somefield = 'somevalue'
if hasattr(a, 'somefield'): 
...   print("T")
... 
T
if 'somefield' in a: 
...   print("T")
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'Object' is not iterable
```
Am I missing something?

Reply:
Oh right, thought args is a dictionary, it's actually a `Namespace` object.